web/cgi-bin/horas/horas.pl

```diff
@@ -212,11 +212,7 @@ sub resolve_refs {
     elsif ($line =~ /^\s*\!\!(.*)/) {  
       my $l = $1;
       my $suffix = '';
-      if ($l =~ /\{.*?\}/) {
-        $l =~ s/(\{.*?\})//; 
-  	    $suffix = $1;
-		    $suffix = setfont($smallblack, $suffix); 
-      }
+      if ($l =~ s/\{[^:].*?\}//) {$suffix = setfont($smallblack, $&);}
```

Comment:
Please avoid `$&` (and ``$` `` and `$'`).  The server is running perl 5.10 and the use of those variables makes matches more expensive on old perls.  The generic way around this is to use an explicit capture group around the whole pattern.